feature-configs/cn-ran-overlays/ran-profile/is_ready.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Check for updated worker-cnf node
+
+oc wait mcp/worker-cnf --for condition=updated --timeout 1s
+
+
+# Check for real-time kernel.
+
+WORKER_NODE=$(oc get nodes --selector="node-role.kubernetes.io/worker-cnf" --output="name")
+
+echo "worker-cnf node is: ${WORKER_NODE}"
+
+oc describe "${WORKER_NODE}" | grep "Kernel Version:.*\.rt[0-9]*\..*\.x86_64"
```

Comment:
I think that checking just the kernel is not enough.
IIRC - there are 2 reboots
one for PAO 
and another for SRIOV

not sure if we even control the order
but need to check for both

you could actually search for the vfs in the same describe command output

Reply:
According to Vitaly, the necessary reboots are done when the worker reports 'updated', which happens before this. I verification of the RT kernel as an additional, post-update sanity check--something that's a necessary but not sufficient indication of a successful update.

Reply:
sriov daemon can reboot the node on it's own
https://github.com/openshift/sriov-network-operator/blob/master/pkg/daemon/daemon.go#L513

Reply:
But that still wouldn't cause the previous check--that the worker-cnf node has completed its update--to pass, right?

In the QE pipeline, this profile deployment and subsequent verification isn't performed until after all the CNF operators, such as sriov, have been applied and verified.

Reply:
I'm going to move this Kernel-version check out of the is_ready.sh script and into the QE Jenkins pipeline along with other tests that go beyond just ensuring the update is done.

Reply:
I have removed this check and will do it as part of the QE pipeline's other checks & tests.

Reply:
yes, but the problem remian.
I verified this.
sriov operator will reboot the node in some cases, which are:
- mlx: every time to change the max number of VFs
- intel: just to enable iommu in the linux cmd for the first time
and we can't know/control the order
so I think, for cn-ran, is_ready,sh should do 2 things:
1. wait for the mcp (as you already did)
2. verify vfs exists in oc describe node

for #2 there, maybe there's a better way? @SchSeba

Reply:
Oh, duh, I missed the vfs part. Thanks. Investigating...